host_core/lib/host_core/control_interface/server.ex

```diff
@@ -194,7 +194,8 @@ defmodule HostCore.ControlInterface.Server do
   # %{"actor_ref" => "bindle://example.com/echo/0.12.0", "host_id" => "Nxxxx", "count" => 4}
   defp handle_request({"cmd", _host_id, "la"}, body, _reply_to) do
     with {:ok, start_actor_command} <- Jason.decode(body),
-         actor_ref <- Map.get(start_actor_command, "actor_ref") do
+         actor_ref <- Map.get(start_actor_command, "actor_ref"),
+         true <- actor_ref != nil do
```

Comment:
There's actually a builtin function `is_nil` that you can use for this too 😄 https://hexdocs.pm/elixir/1.12/Kernel.html#is_nil/1